site/docs/guide/middleware.md

```diff
@@ -253,7 +253,7 @@ Middleware objects are simply a wrapper for middleware functions.
 They are mostly used internally, but can sometimes also help third-party libraries, or be used in advanced use cases, such as with [Composer](https://deno.land/x/grammy/mod.ts?s=Composer):
 
 ```ts
-const bot = new Bot("<token>");
+const bot = new Bot(""); // <-- put your bot token between the ""
```

Comment:
I don't like this. The topic on this page is completely unrelated to getting your bot up and running. The comment is useless information here, and it distracts from what we actually want to say. What's consistency worth if applied in places it doesn't belong?